torch_geometric/nn/models/captum.py

```diff
@@ -176,59 +220,65 @@ def to_captum_input(x: Union[Tensor, Dict[EdgeType, Tensor]],
-        else:
-            inputs = [x.unsqueeze(0), _to_edge_mask(edge_index).unsqueeze(0)]
-        additional_forward_args.append(edge_index)
+        inputs = []
+        if "node" in mask_type:
+            inputs.append(_expand_input(x, n_samples))
+        if "edge" in mask_type:
+            inputs.append(_expand_input(_to_edge_mask(edge_index), n_samples))
+        if mask_type == "edge":
+            additional_forward_args.insert(0, x)
 
     elif isinstance(x, Dict) and isinstance(edge_index, Dict):
         node_types = x.keys()
         edge_types = edge_index.keys()
         inputs = []
         if mask_type == "node":
             for key in node_types:
-                inputs.append(x[key].unsqueeze(0))
+                inputs.append(_expand_input(x[key], n_samples))
         elif mask_type == "edge":
             for key in edge_types:
-                inputs.append(_to_edge_mask(edge_index[key]).unsqueeze(0))
-            additional_forward_args.append(x)
+                inputs.append(
+                    _expand_input(_to_edge_mask(edge_index[key]), n_samples))
+            additional_forward_args.insert(0, x)
         else:
             for key in node_types:
-                inputs.append(x[key].unsqueeze(0))
+                inputs.append(_expand_input(x[key], n_samples))
             for key in edge_types:
-                inputs.append(_to_edge_mask(edge_index[key]).unsqueeze(0))
-        additional_forward_args.append(edge_index)
+                inputs.append(
+                    _expand_input(_to_edge_mask(edge_index[key]), n_samples))
 
     else:
         raise ValueError(
             "'x' and 'edge_index' need to be either"
             f"'Dict' or 'Tensor' got({type(x)}, {type(edge_index)})")
-    additional_forward_args.extend(args)
     return tuple(inputs), tuple(additional_forward_args)
 
 
 def captum_output_to_dicts(
-    captum_attrs: Tuple[Tensor], mask_type: str, metadata: Metadata
-) -> Tuple[Optional[Dict[NodeType, Tensor]], Optional[Dict[EdgeType, Tensor]]]:
-    r"""Convert the output of `Captum.ai <https://captum.ai/>`_ attribution
-    methods which is a tuple of attributions to two dictonaries with node and
-    edge attribution tensors. This function is used while explaining
-    :obj:`HeteroData` objects. See :obj:`torch_geometric.nn.to_captum_model`
-    for example usage.
+    captum_attrs: Tuple[Tensor],
+    mask_type: str,
+    metadata: Metadata,
+) -> Tuple[Optional[List[Dict[NodeType, Tensor]]], Optional[List[Dict[
+        EdgeType, Tensor]]]]:
+    """Convert the output of `Captum.ai <https://captum.ai/>`_ attribution
+    methods which is a tuple of attributions to dictonaries with with node and
+    edge attribution tensors. If more than one object is explained, the
+    explanations are stored in a list of dictionaries. This function is used
+    while explaining :obj:`HeteroData` objects. See
+    :obj:`torch_geometric.nn.to_captum_model` for example usage.
 
```

Comment:
```suggestion
    :obj:`~torch_geometric.nn.to_captum_model` for example usage.

```

Reply:
What is `~` doing?

Reply:
It will only maintain the method function name in the documentation.